cytosnake/cli/exec/workflow_exec.py

```diff
@@ -8,14 +8,15 @@
 
 import snakemake
 
-from cytosnake.utils.config_utils import load_workflow_path
+from cytosnake.utils.config_utils import load_general_configs
 
 
 def __base_exec(
     workflow_file: str,
     n_cores: Optional[int] = 1,
     unlock: Optional[bool] = False,
     force: Optional[bool] = False,
+    env_manager: Optional[str] = "conda",
```

Comment:
just confirming that you want `conda` as default

Reply:
Correct, the default will be `conda`. If users want to switch to `mamba`, then users need to go to the `configurational.yaml` file